Header header

float64 receiver_gain
gazebo_wifi_plugin/ReceiverRouterInfo[] router_info
